static/src/js/components/CustomWidgetWrapper/CustomWidgetWrapper.jsx

```diff
@@ -45,13 +47,15 @@ const CustomWidgetWrapper = ({
     setShowHelp(false)
   }
 
+  const predeterminedJustification = centered ? 'justify-content-center' : 'justify-content-between'
```

Comment:
The `classnames` library should be used when conditionally adding classes to elements. Search the project for `classNames(` to find a good example